pkg/cmd/grafana-server/main.go

```diff
@@ -87,7 +88,11 @@ func main() {
 
 	err := server.Run()
 
-	server.Exit(err)
+	code := server.Exit(err)
+	trace.Stop()
```

Comment:
you do not need trace.Stop it's called above via  defer on line ~70

Reply:
its only called if profile is enabled.

Reply:
Looking at this.. that also seems messy.